Modelica/Blocks/Logical.mo

```diff
@@ -20,6 +20,20 @@ the output is <strong>false</strong>.
 </html>"));
   end And;
 
+  block MultiAnd "Logical and of Boolean vector elements"
+    extends Modelica.Blocks.Interfaces.PartialBooleanMISO;
+  equation
+    y = Modelica.Math.BooleanVectors.andTrue(u);
+    annotation (Icon(coordinateSystem(initialScale=0.1),
+      graphics={ Text(extent={{-90,40},{90,-40}},textString="and")}),
+      Documentation(info="<html>
+<p>
+The output is <strong>true</strong> if all elements of the input vector are <strong>true</strong>, otherwise
```

Comment:
Must also mention that the input vector must be non-empty:
```suggestion
The output is <strong>true</strong> if the input vector is not empty and all elements of the input vector are <strong>true</strong>, otherwise
```

Reply:
Wouldn't it be a lot better to handle the case where the input vector is empty (in the usual consistent mathematical way) instead of documenting the exception?

Reply:
Since `andTrue` (which is used) is the normal implementation of `and` for an array and has the description: 

> Returns true if all elements of the Boolean input vector b are true.

I don't see a problem with the previous formulation.

Reply:
No, the whole problem is that this freaking block uses `allTrue` instead of `andTrue`.

Reply:
Sorry about the confusion.  I'll close this and move the suggestion to `MultiNand` where it belongs.